config/runtime/server.go

```diff
@@ -637,15 +638,16 @@ func configureAccessControls(conf *config.Couper, confCtx *hcl.EvalContext, log
 				}
 
 				jwt, err = ac.NewJWT(&ac.JWTOptions{
-					Algorithm:      jwtConf.SignatureAlgorithm,
-					Claims:         jwtConf.Claims,
-					ClaimsRequired: jwtConf.ClaimsRequired,
-					Key:            key,
-					Name:           jwtConf.Name,
-					RolesClaim:     jwtConf.RolesClaim,
-					RolesMap:       jwtConf.RolesMap,
-					ScopeClaim:     jwtConf.ScopeClaim,
-					Source:         ac.NewJWTSource(jwtConf.Cookie, jwtConf.Header, jwtConf.TokenValue),
+					Algorithm:             jwtConf.SignatureAlgorithm,
```

Comment:
possible improvement:
* create `&ac.JWTOptions{...}` with members used in **both** cases outside the `if`
* set the members specific to the case (`if` / `else`) and then call `ac.NewJWTFromJWKS()` / `ac.NewJWT()` with the modified `JWTOptions`

So there will be a lower probability that a new common member is added in only one of the cases